vpd-manager/src/parser.cpp

```diff
@@ -59,8 +60,41 @@ types::VPDMapVariant Parser::parse()
 int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
 {
     int l_bytesUpdatedOnHardware = constants::FAILURE;
+
+    // A lambda to extract Record : Keyword string from i_paramsToWriteData
+    auto l_keyWordIdentifier =
+        [](const types::WriteVpdParams& i_paramsToWriteData) -> std::string {
```

Comment:
instead taking as const ref for i_paramsToWriteData, we could directly used by mentioning in input clause ?
l_keyWordIdentifier()  is used for trace/PEL, for which vpd file path is missing in this.
can you add 'm_vpdFilePath' as well ?

Reply:
@branupama 
PELs are logged for reboot block/unblock failures. Why it should log details of VPD path?
If write fails then there is a log which gives that detail. Is there a requirement of duplicate data here?

Reply:
ok, as record and keyword values are passed, I thought vpd path is missed out.